net/acme/files/run.sh

```diff
@@ -107,6 +132,10 @@ post_checks()
         /etc/init.d/uhttpd reload
         UHTTPD_LISTEN_HTTP=
     fi
+
+    if [ -e /etc/init.d/nginx ] && [ "$NGINX_WEBSERVER" -eq 1 ]; then
```

Comment:
This should probably trigger if *either* nginx was previously stopped, *or* update_nginx is set to 1. Otherwise, the new certificate won't be applied in the case where nginx is only listening on port 443.

Reply:
think i will modify it to check if nginx is installed and if update_nginx is 1...

Reply:
It should probably be either or; if nginx is killed during setup, it should be restarted regardless of whether the config is updated...